.env.example

```
@@ -9,9 +9,12 @@ NODE_ENV=development
 # Database Configuration
 DATABASE_PATH=./data/crewai_enhanced.db
 
-# Ollama Configuration
-OLLAMA_BASE_URL=http://localhost:11434
-OLLAMA_TIMEOUT=30000
+# Llama.cpp Configuration
+LLAMA_CPP_PATH=/path/to/llama.cpp/build/bin/llama-cli
+LLAMA_MODEL_PATH=./models/Llama-3.2-3B-Instruct-Q4_K_M.gguf
+LLAMA_GPU_LAYERS=0
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The LLAMA_GPU_LAYERS key should go before the LLAMA_MODEL_PATH key

(UnorderedKey)